File: assignments/functions/only_float.py
def only_float(a, b):
	if(a == int(a) and b == float(b)):
		return 1
	if(a == float(a) and b == int(b)):
		return 1
	if(a == float(a) and b == float(b)):
		return 2


number1 = float(input('Enter a number: '))
number2 = float(input('Enter a number: '))
total = only_float(number1, number2)

print(total)

# this is your test will always pass as you already cast ur input to float

File: assignments/javac/OnlyFloat.java
import java.util.Scanner;
public class OnlyFloat{
	

	public static int decide(double number1, double number2){

	if( number1 == (float) number1 && number2 == (float) number2) return 2;
	if( number1 == (int) number1 && number2 == (float) number2) return 1;
	if( number1 == (float) number1 && number2 == (int) number2) return 1;
	else return 0;
	}

	

public static void main(String []args){
	Scanner input = new Scanner(System.in);

	System.out.print("Enter a number: ");
	double number1 = input.nextInt();

	System.out.print("Enter a number: ");
	double number2 = input.nextInt();

	double total = decide(number1, number2);
	System.out.print(total);

}
}